contracts/SportMarkets/SportsAMM.sol

```diff
@@ -564,7 +553,30 @@ contract SportsAMM is Initializable, ProxyOwned, PausableUpgradeable, ProxyReent
         uint additionalSlippage,
         address collateral
     ) public nonReentrant whenNotPaused {
-        _buyFromAMMWithDifferentCollateral(market, position, amount, expectedPayout, additionalSlippage, collateral);
+        _buyFromAMMWithDifferentCollateral(market, position, amount, expectedPayout, additionalSlippage, collateral, false);
+    }
+
+    /// @notice Buy amount of position for market/game from AMM using ETH
+    /// @param market The address of the SportPositional market of a game
+    /// @param position The position (home/away/draw) to buy from AMM
+    /// @param amount The position amount to buy from AMM
+    /// @param expectedPayout The amount expected to pay in sUSD for the amount of position. Obtained by buyAMMQuote.
+    /// @param additionalSlippage The slippage percentage for the payout
+    /// @param collateral The address of the collateral used
+    /// @param _referrer who referred the buyer to SportsAMM
+    function buyFromAMMWithEthAndReferrer(
+        address market,
+        ISportsAMM.Position position,
+        uint amount,
+        uint expectedPayout,
+        uint additionalSlippage,
+        address collateral,
```

Comment:
collateral variable makes little sense in the usage

Reply:
Its used to get the quote and check for slippage.

Reply:
might be obsolete as it doesnt match the current code base